### DISCUSSION TITLE
Code Coverage Bug

### DISCUSSION THREAD

**Discussion by cesarneurored** [forcedotcom/salesforcedx-vscode#5532] [General]
With the new update of the extension the code coverage doesn't work
![image](https://github.com/forcedotcom/salesforcedx-vscode/assets/150020978/0b059f66-6f62-4c44-9c72-48eb35a3fb6b)

**Comment by rin183461** [comment]
Similar error when I attempt to check code coverage from VSCode as well:
![image](https://github.com/forcedotcom/salesforcedx-vscode/assets/79720055/f89e5502-dbd2-40af-a3f7-48bebdce9aa4)
